package/pom.xml

```diff
@@ -128,6 +150,78 @@
           </filesets>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.3.0</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>extra-enforcer-rules</artifactId>
+            <version>1.7.0</version>
+          </dependency>
+        </dependencies>
+        <executions>
+          <execution>
+            <id>enforce-versions</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>3.6.3</version>
+                </requireMavenVersion>
+              </rules>
+            </configuration>
+          </execution>
+          <execution>
+            <id>enforce-ban-duplicate-classes</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <banDuplicateClasses>
+                  <ignoreClasses>
+                    <!-- The UnusedStubClass from Spark -->
+                    <ignoreClass>org.apache.spark.unused.UnusedStubClass</ignoreClass>
+                    <!-- jdo -->
+                    <ignoreClass>javax.jdo.*</ignoreClass>
+                    <!-- log4j -->
+                    <ignoreClass>org.apache.commons.logging.*</ignoreClass>
+                    <!-- hive -->
+                    <ignoreClass>org.apache.hadoop.hive.*</ignoreClass>
+                    <!-- celeborn -->
+                    <ignoreClass>org.apache.spark.sql.execution.columnar.ByteBufferHelper</ignoreClass>
+                    <ignoreClass>org.apache.spark.sql.execution.columnar.ByteBufferHelper$</ignoreClass>
+                    <!-- The overridden class list by Gluten. Carefully add entries to this list only when you knew exactly what is going to happen -->
+                    <ignoreClass>org.apache.spark.rdd.EmptyRDD</ignoreClass>
+                    <ignoreClass>org.apache.spark.sql.hive.execution.HiveFileFormat</ignoreClass>
+                    <ignoreClass>org.apache.spark.sql.hive.execution.HiveFileFormat$$$$anon$1</ignoreClass>
```

Comment:
I download spark-3.2.2-bin-hadoop2.7.tgz find this class may seem different, could you give more notes on how to add entries? @zhztheplayer 

```
[root@host]# jar -tf spark-hive_2.12-3.2.2.jar | grep HiveFile
org/apache/spark/sql/hive/execution/SaveAsHiveFile.class
org/apache/spark/sql/hive/execution/HiveFileFormat.class
org/apache/spark/sql/hive/execution/HiveFileFormat$$anon$1.class
```

Reply:
It was said that you don't have to do anything but just to use spark-submit / spark-shell `--jar` to let gluten jar to be loaded before other jars.

The list is just an exclusion list to let use know how many classes gluten overrides from vanilla Spark.